### PR TITLE
Copy production scaling config to staging prior to loadtest

### PR DESCRIPTION
**Pull request by sakisv**
---

🚨⚠️ This will be deployed automatically all the way to production when you click merge ⚠️🚨

For more information, including how to check this deployment on preview or staging first before it goes to production, see our [team wiki section on continuous deployment](https://github.com/alphagov/notifications-manuals/wiki/Deploying-with-concourse#continuous-deployment)

### CODE REVIEW

**Review by CrystalPea — APPROVED** [review]
LGTM

**Comment by sakisv** [comment]
Waiting for confirmation that the isolation segment is ready before merging this, as it will scale up all the apps

**Pull request merged**
